man/ocaml.1

```diff
@@ -70,6 +70,9 @@ The following command-line options are recognized by
 .B \-absname
 Show absolute filenames in error messages.
 .TP
+.B \-no-absname
+Do not Show absolute filenames in error messages.
```

Comment:
Lowercase 's' wanted on show here (and in the other two files):
```suggestion
Do not show absolute filenames in error messages.
```

Reply:
There is also the slight issue that the negation is inexact: if ocamlc is called on a an absolute file name, `ocamlc -no-absname /absolute/path/file.ml` will show this absolute filename without trying to make it relative to the current working directory.
I would propose to make that point more obvious with:
```
Do not normalize filenames to absolute filenames in error messages.
```

Reply:
okay, noted. Corrections will be made.